### DISCUSSION TITLE
Amasty Advance Review

### DISCUSSION THREAD

**Discussion by hungdq123** [breezefront/community#122] [3. Integrations]
The module Amasty Advance Review do not work when I enable Breeze module.
Hope your team can integrate it!
Thanks

**Comment by jihadbek** [comment]
Well , i am using mageplaza better reviews and i am facing issues with it anyway .. breeze could work on amasty review extension i would switch to it as well when i go with breeze .

**Comment by dimdimich** [comment]
Thank you for your interest. Please contact Amasty also. They should know that more and more people are interested in Breeze integration. We will also check from our side.

**Reply by jihadbek** [reply]
done already

**Comment by dimdimich** [comment]
> The module Amasty Advance Review do not work when I enable Breeze module. Hope your team can integrate it! Thanks

Please contact also Amasty and send link to this discussion. More people will ask about integration - more chances that Amasty will add it from their side.

**Comment by georgetze** [comment]
This module doesn't work with breeze.
